src/sentry/utils/snuba_rpc.py

```diff
@@ -82,11 +85,12 @@ def rpc(req: SnubaRPCRequest, resp_type: type[RPCResponseType]) -> RPCResponseTy
 
         cls = req.__class__
         class_name = cls.__name__
+        endpoint_name = ENDPOINT_NAMES.get(class_name, class_name)
```

Comment:
I think we talked about this but it'd be nice if we can clean this up interface a little into something like
```python
rpc_response = snuba_rpc.table_rpc(rpc_request)
```

And the response type as well as the endpoint is hard coded within the `table_rpc` function which then calls this `rpc` function with the endpoint name.

Reply:
Ah i forgot, thanks for the reminder. I'll do that in a follow up